### PR TITLE
Final touches for 1.13.0

### PR DESCRIPTION
**Pull request by juliangieseke**
## Description

This PR contains a couple last minute things we'd like to have in 1.13.0

1. Disable New Relic now so that we dont have to disable it after MWT
2. Add latest chinese translations
3. Add small update to status codes for SDK Service status (some have been changed to `3xx`, because marathon is killing services report `4xx` status codes.)

## Testing

You can smoke test manually be checking the new Chinese translation strings.

### CODE REVIEW

File: locale/zh/messages.json [hunk 1]
```diff
@@ -1,1171 +1,1171 @@
...
-  "{userLabel} will be removed from the {groupLabel} group.": "{userLabel} 将从 {groupLabel} 组移除。",
-  "{userName} will be removed from {groupLabel}.": "{userName} 将从 {groupLabel} 移除。",
-  "{watching} out of {totalFound}": "{watching} 用完 {totalFound}"
+{
+  "(AT BEGINNING OF FILE)": "（运行文件时）",
+  "(Clear)": "（清除）",
+  "(No data)": "（无数据）",
+  "(Optional. Default: 20): Number of seconds after which a health check is considered a failure regardless of the response.": "（可选。默认值：20）：运行状况检查被视为失败（无论响应如何）后的秒数。",
+  "(Optional. Default: 3): Number of consecutive health check failures after which the unhealthy instance should be killed. HTTP & TCP health checks: If this value is 0, instances will not be killed if they fail the health check.": "（可选。默认值：3）：不健康实例应被杀死之后运行状况检查连续失败的次数。HTTP 和 TCP 运行状况检查：如果此值为 0，那么尽管实例执行运行状况检查失败，也不会被杀死。",
+  "(Optional. Default: 300): Health check failures are ignored within this number of seconds or until the instance becomes healthy for the first time.": "（可选。默认值：300）：运行状况检查失败在此秒数内或直到实例
```
Comment (mperrotti) [inline locale/zh/messages.json]: In the new translations, "Arg"/"Args" is all caps. Is this intentional?

**Comment by DanielMSchmidt** [comment]
I found some parts that seem to be untranslated. I am unfamiliar if this is normal, so I'm just posting this here: 

Bildschirmfoto 2019-04-17 um 16 32 41
Bildschirmfoto 2019-04-17 um 16 33 18
Bildschirmfoto 2019-04-17 um 16 33 25

**Comment by GeorgiSTodorov** [comment]
I get this.
Снимка от 2019-04-17 18-11-27

**Comment by DanielMSchmidt** [comment]
@GeorgiSTodorov Your private plugins might be on a diffrent branch then 1.13

**Comment by GeorgiSTodorov** [comment]
They need to be on 1.13 and not master?

File: locale/zh/messages.json (hunk 1 above)
Comment (juliangieseke) [reply locale/zh/messages.json]: Think so… They did the same with some other things.

**Comment by GeorgiSTodorov** [comment]
I think the testing instructions need to be updated if the private plugins need to be on 1.13 and not on master.

**Comment by nLight** [comment]
@GeorgiSTodorov that is was usually implied by testing anything but master

**Comment by nLight** [comment]
Yes, we still have some things  not translated, we aim for _more coverage_ not _100% coverage_

**Comment by GeorgiSTodorov** [comment]
@nLight , where does it say to test anything but master?

**Comment by nLight** [comment]
@GeorgiSTodorov it doesn't and I called that out saying "implies". By that I mean `dcos-ui` and `plugins` branches should be synchronised. Sometimes it just works without them being in sync but usually not when testing `release/*`.

**Comment by juliangieseke** [comment]
@DanielMSchmidt we dont transl;ate API reponses yet, so its okay to only have ~80% translated.
@GeorgiSTodorov as Dima said, when testing PRs its always good to have plugins on the correct branch. we might want to add it to plugins readme tho.

**Comment by GeorgiSTodorov** [comment]
It's easy to miss that we are merging this to 1.13 and not to master and the fact that 1.13 is the correct branch for the private plugins is not something obvious.

**Comment by GeorgiSTodorov** [comment]
Also, I am not sure what's the point of smoke testing this, since we don't know which parts are supposed to me translated and we can't notice a wrong translation.

**Pull request merged**